### DISCUSSION TITLE
Can we order by some of the value of  relation field?

### DISCUSSION THREAD

**Discussion by shtse8** [prisma/prisma#14173] [Q&A] [answered]
Reference to the issue: https://github.com/prisma/prisma/issues/5008#issuecomment-768444693

Schema:
```prisma
model Author {
  id Int @id
  name String
  posts Post[]
}

model Post {
  id Int @id
  title String
  author Author
  views Int
}
```

I want to find the top 1 author with a query like this.
```
prisma.author.findFirst({
  orderBy: {
    articles: {
      views: {
        _sum: 'desc'
      }
    }
  }
})
```

Is it possible?

**Comment by ruheni** [comment, marked as answer]
Hey, Prisma currently doesn't support filtered relation count aggregate. We have an open [GitHub issue](https://github.com/prisma/prisma/issues/6570) on a similar use case, and we would appreciate it if you would upvote it and leave a comment with your use case to help our Product team prioritize the feature request. 🙂  

The existing workaround would be using a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
